### PR TITLE
LPS-139865 Fix PagintationBarTag server side rendering

### PR DESCRIPTION
**Pull request by julien**
The PagintationBarTag wasn't rendering exactly the same html
as it's corresponsing Clay component, this change fixes that.

**Test plan**:
- Fetch this branch
- Deploy the `frontend-taglib-clay` OSGi module
- Assert that the PaginationBarTag renders the same html
  as the Clay component

### CODE REVIEW

**Review by javierdearcos — APPROVED** [review]
LGTM

I think it is even clearer with the method extraction and with the totalItems new name.

My doubt now is about the alphabetical order on _processItem method parameters, that usually Brian wants to enforce. But I would leave it like that and try.

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java [hunk 1]
```diff
@@ -265,88 +270,135 @@ protected int processStartTag() throws Exception {
...
 
-		for (int i = ellipsisCount + 1; i < (totalPages + 1); i++) {
-			jspWriter.write("<li class=\"page-item");
+		return SKIP_BODY;
+	}
 
-			if (i == _activePage) {
-				jspWriter.write(" active");
-			}
+	private void _processEllispis(PageContext pageContext) throws Exception {
```
Comment (jbalsas) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java:358]: There's a small typo here, was this meant to be `_processEllipsis`? see the `sp` transposition.

Comment (julien) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java:358]: Thanks for the catch @jbalsas :facepalm: I always type ellipsis wrong

**Comment by john-co** [comment]
Reviewed. Test failures are unrelated. :heavy_check_mark: 
- PortalSmokeUpgrade#ViewPortalSmokeArchive621021 - known flaky isssue cc/ @vicnate5
```
LIFERAY_ERROR: bundle com.liferay.portal.properties.swapper:5.0.4 (308)[com.liferay.portal.properties.swapper.internal.DefaultGuestGroupLogoSwapper(10050)] : The activate method has thrown an exception 
```

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:relevant

**Comment by nhpatt** [comment]
Send directly here: https://github.com/brianchandotcom/liferay-portal/pull/108140

**Pull request closed without merging**